frontends/PyCDE/src/bit_pat.py

```diff
@@ -0,0 +1,107 @@
+# ===============================================================================
```

Comment:
I'm sorry but we cannot allow any code with any license other than apache 2.0. If you want to include this in CIRCT proper, you'll need to contact the Magma guys to see if they're willing to re-license it to Apache 2.0 w/ LLVM exceptions.

Reply:
they don't have to relicense the entire project, just provide you a copy of that file under the required license.